debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -636,9 +636,20 @@ public Boolean isFullColummnScanRequired() {
         return getConfig().getBoolean(SNAPSHOT_FULL_COLUMN_SCAN_FORCE);
     }
 
+    private static int validateColumnBlacklist(Configuration config, Field field, Field.ValidationOutput problems) {
+        String blacklist = Configuration.logFallbackStringProperty(config, COLUMN_INCLUDE_LIST, COLUMN_WHITELIST);
+        String whitelist = Configuration.logFallbackStringProperty(config, COLUMN_EXCLUDE_LIST, COLUMN_BLACKLIST);
+
+        if (whitelist != null && blacklist != null) {
+            problems.accept(COLUMN_BLACKLIST, blacklist, COLUMN_INCLUDE_LIST_ALREADY_SPECIFIED_ERROR_MSG);
```

Comment:
Can we split up `COLUMN_INCLUDE_LIST_ALREADY_SPECIFIED_ERROR_MSG` into two, one for include list and one for the legacy term? The reason being, that the legacy term shouldn't show up for a wrong usage of the new options. Same for the other similar messages.